tests/regression/11-heap/09-no_strong_update_not_unique_thread.c

```diff
@@ -0,0 +1,33 @@
+// PARAM: --set ana.malloc.unique_address_count 2
+
+// Strong updates are not possible in thread because it is not unique.
```

Comment:
How about with mallocFresh analysis also enabled? These alloc variables are created thread-local and never escape, so theoretically they should be strongly updateable.

Reply:
The corresponding variables are still created as globals here, so freshness does not help here.

Reply:
Oh right, mallocFresh analysis only affects race checking, but not anything value-based, which I suppose it also could. Actually the freshness stuff could become part of escape analysis, but there was some issue at the time which prevented that (considering everything non-escaped to be fresh in general).

Reply:
Yes, in the future, we should definitely make that change. I'd suggest doing it in a different PR though.